src/mono/netcore/System.Private.CoreLib/src/ILLink/ILLink.Substitutions.wasm.xml

```diff
@@ -1,7 +1,14 @@
 <linker>
   <assembly fullname="System.Private.CoreLib">
     <type fullname="System.Numerics.Vector">
-      <method signature="System.Boolean get_IsHardwareAccelerated()" body="stub" value="false" /> 
+      <method signature="System.Boolean get_IsHardwareAccelerated()" body="stub" value="false" />
+    </type>
+    <type fullname="System.Environment">
+      <method signature="System.Boolean get_IsSingleProcessor()" body="stub" value="true" />
+      <method signature="System.Int32 get_ProcessorCount()" body="stub" value="1" />
+    </type>
+    <type fullname="System.Runtime.CompilerServices.RuntimeFeature">
+      <method signature="System.Boolean get_IsDynamicCodeCompiled()" body="stub" value="false" />
     </type>
```

Comment:
What does this conditional mean ? Is running with the mono interpreter means 'Compiled' ?

Reply:
Compiles mean JIT in this context so it should be false for AOT, Interpreter